src/main/java/edu/illinois/cs/dt/tools/utility/iFixPlusPlugin.java

```diff
@@ -0,0 +1,744 @@
+package edu.illinois.cs.dt.tools.utility;
+
+import com.google.gson.Gson;
+import com.reedoei.eunomia.io.files.FileUtil;
+import edu.illinois.cs.dt.tools.minimizer.MinimizeTestsResult;
+import edu.illinois.cs.dt.tools.minimizer.PolluterData;
+import edu.illinois.cs.dt.tools.runner.InstrumentingSmartRunner;
+import edu.illinois.cs.dt.tools.runner.data.DependentTest;
+import edu.illinois.cs.dt.tools.runner.data.DependentTestList;
+import edu.illinois.cs.testrunner.configuration.Configuration;
+import edu.illinois.cs.testrunner.data.results.TestRunResult;
+import edu.illinois.cs.testrunner.mavenplugin.TestPlugin;
+import edu.illinois.cs.testrunner.mavenplugin.TestPluginPlugin;
+import edu.illinois.cs.testrunner.runner.Runner;
+import edu.illinois.cs.testrunner.runner.RunnerFactory;
+import org.apache.commons.io.FileUtils;
+import org.apache.maven.project.MavenProject;
+import org.w3c.dom.Node;
+import scala.util.Try;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.BufferedReader;
+import java.io.PrintWriter;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.io.UnsupportedEncodingException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import java.util.Set;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.HashSet;
+
+import org.xmlunit.builder.DiffBuilder;
+import org.xmlunit.diff.Comparison.Detail;
+import org.xmlunit.diff.DefaultNodeMatcher;
+import org.xmlunit.diff.Diff;
+import org.xmlunit.diff.Difference;
+import org.xmlunit.diff.ElementSelectors;
+
+public class iFixPlusPlugin extends TestPlugin {
+    private Path replayPath;
+    private Path replayPath2;
```

Comment:
Can we use a better name than `replayPath2` as to distinguish what it is actually supposed to represent?